client/my-sites/plugins/manage-site-plugins-dialog/index.jsx

```diff
@@ -46,32 +34,16 @@ export const ManageSitePluginsDialog = ( { isVisible, onClose, plugin } ) => {
 			onClose={ onClose }
 			shouldCloseOnEsc
 		>
-			<PluginSiteList
-				className="manage-site-plugins-dialog__plugin-list"
-				title={ translate( 'Installed on %d site', 'Installed on %d sites', {
-					comment: 'header for list of sites a plugin is installed on',
-					args: [ sitesWithPlugin.length ],
-					count: sitesWithPlugin.length,
-				} ) }
+			<SitesWithInstalledPluginsList
 				sites={ sitesWithPlugin }
+				isLoading={ isLoading }
 				plugin={ plugin }
-				showAdditionalHeaders
-			/>
-
-			<PluginSiteList
-				className="manage-site-plugins-dialog__plugin-list"
-				title={ translate( 'Available sites' ) }
-				sites={ availableOnSites }
-				plugin={ plugin }
-				billingPeriod={ billingPeriod }
 			/>
 
-			<PluginSiteList
-				className="manage-site-plugins-dialog__plugin-list"
-				title={ translate( 'Upgrade needed' ) }
-				sites={ upgradeNeededSites }
```

Comment:
A question out of curiosity – do you know what was the purpose of this section?

Reply:
We have 2 different sections which did the same thing, now we have one section which says `Available On`